snowpenguin/django/recaptcha3/fields.py

```diff
@@ -25,42 +25,47 @@ def __init__(self, attrs=None, *args, **kwargs):
         super(ReCaptchaField, self).__init__(*args, **kwargs)
 
     def clean(self, values):
-
-        # Disable the check if we run a test unit
-        if os.environ.get('RECAPTCHA_DISABLE', None) is not None:
-            return values[0]
+        # Mock response if we run disabled
+        json_response = {
+            'success': True,
+            'score': 0.5,
+            'action': 'homepage',
+            'challenge_ts': '2021-01-24T20:00:42Z',
```

Comment:
I'm not a big fan of mocked responses outside test units files. Also because it fails if somebody uses a lower score threshold than 0.5. You must at least to place the `if bool(json_response['success']):` block inside `if os.environ.get('RECAPTCHA_DISABLE', None) is None:`

You can also just return the mocked json response early if the `RECAPTCHA_DISABLE` env variable has a value. what do you think?

Reply:
Returning 0.5 _through_ the scoring logic allows the developer to be able to trigger either success or failure in their unit tests by changing `score_threshold`. An improvement might be to set `RECAPTCHA_DISABLE` to the desired score value, such as `"0.5"`? Perhaps allowing `"False"` to trigger a `success: False` server error?

The only way I can see to completely avoid this 'mocked response outside of unit tests' would be to put the entire server response inside `RECAPTCHA_DISABLE`, ie. allow the user to set:
```RECAPTCHA_DISABLE = "{'success': True, 'score': 0.5, 'action': 'homepage', 'challenge_ts': '2021-01-24T20:00:42Z'}"
```
Would need to opportunistically `json.loads` the env var, else do... what?